modules/security-context-constraints-about.adoc

```diff
@@ -87,6 +97,16 @@ endif::[]
 |`nonroot`
 |Provides all features of the `restricted` SCC, but allows users to run with any non-root UID. The user must specify the UID or it must be specified in the manifest of the container runtime.
 
+|`nonroot-v2`
+| Like the `nonroot` SCC, but with the following differences:
+
+* `ALL` capabilities are dropped from containers.
+* The `NET_BIND_SERVICE` capability can be added explicitly.
```

Comment:
@stlaz can I use `NET_BIND_SERVICE ` catability? Should not I drop ALL or leave empty? We need to better clarifies this one